vspace/src/main/webapp/WEB-INF/views/staff/modules/module.jsp

```diff
@@ -39,18 +39,43 @@ $(document).ready(function($) {
 				}
 		});
 	});
-});	
+	$(".sequences").on("click", function(e) {
+		var sequenceId = this.id;
+		$.ajax({
+			type: "POST",
+			url: "<c:url value="/staff/module/${module.id}/"/>" +sequenceId +'?${_csrf.parameterName}=${_csrf.token}',
+			async: true,
+			success: function(response) {
+				$('#message').html("Start sequence updated successfully");
```

Comment:
if you use ajax here, you need to return json or at least an API response in the controller.